components/input/input.css

```diff
@@ -0,0 +1,59 @@
+.ipt {
+    display: flex;
+    flex-direction: column;
+}
+
+.ipt-box {
+    color: var(--text-color);
+    font-size: 1.2rem;
+    line-height: 20px;
+    margin: 0 20px 10px;
```

Comment:
since you are making your page responsive, i would suggest you to use rems here

Reply:
okay